lib/export_tables_to_cloud_storage.rb

```diff
@@ -0,0 +1,66 @@
+require 'google/cloud/storage'
+require 'fileutils'
+
+class ExportTablesToCloudStorage
+  TABLES = %w[
+      Vacancy
+      AlertRun
+      AuditData
+      DetailedSchoolType
+      GeneralFeedback
+      Leadership
+      PayScale
+      Region
+      School
+      SchoolType
+      Subject
+      Subscription
+      TransactionAuditor
+      User
+      VacancyPublishFeedback
+  ]
+
+  attr_reader :bucket
+
+  def initialize(storage: Google::Cloud::Storage.new)
+    @bucket = storage.bucket(ENV['CLOUD_STORAGE_BUCKET'])
+  end
+
+  def run!
+    export_csv_files
+    upload_to_google_cloud_storage
+    remove_csv_folder
+  end
+
+  private
+
+  def export_csv_files
+    Dir.mkdir(Rails.root.join('tmp/csv_export')) unless File.exist?(Rails.root.join('tmp/csv_export'))
+
+    TABLES.each do |table|
+      file = Rails.root.join("tmp/csv_export/#{table.underscore}.csv")
+      records = table.constantize.all
+
+      next unless records.any?
+
+      CSV.open(file, 'w') do |csv|
```

Comment:
:+1: this is a clever way to solve this so it all stays in Ruby.  When I was first thinking about it, I had imagined it as a `.sql` that we ran using the `psql` client.  This keeps it closer to the main code and makes it all very explicit.